listenbrainz/testdata/listens.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
this file should be renamed to something more clear like `influx_listenstore_test_listens.json` or something similar.